pt_miniscreen/core/components/list.py

```diff
@@ -59,6 +61,33 @@ def __init__(
         end_index = self.state["num_visible_rows"] + self.state["top_row_index"]
         self.rows = [self.create_child(Row) for Row in Rows[start_index:end_index]]
 
+    @property
+    def visible_scrollbar(self):
+        return self.state["visible_scrollbar"]
+
+    @visible_scrollbar.setter
+    def visible_scrollbar(self, value):
+        self.state.update({"visible_scrollbar": value})
+
+    @property
+    def distance_to_bottom(self):
+        max_top_row_index = len(self.state["Rows"]) - self.state["num_visible_rows"]
+        return max_top_row_index - self.state["top_row_index"]
+
+    @property
+    def distance_to_top(self):
+        return self.state["top_row_index"]
+
+    @property
+    def can_scroll_down(self):
```

Comment:
These properties assume a scroll distance of 1. It's correct to say if we can scroll down once then we can scroll down but we should probably change them to be functions that also take a distance argument. Later we use these to see if we can scroll in the `scroll_to` function but it doesn't give us enough info (I'll add a comment there with some more detail.)